### ISSUE TITLE
creating a SQL RLS query with IA has UI with low contrast (improve accessibility)

### ISSUE THREAD

**Issue by huilensolis** [supabase/supabase#23245]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

When creating a SQL RLS query with IA, the UI has low contrast. this is difficult to read for people with vision problems and those without.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to a table in your table editor (supabase.com)
2. Click on "add RLS policy"
3. click on "Create policy"
4. on the top of the SQL edit box, click on the "open SQL editor" button
5. ask the IA to write some query
6. see the UI

## Expected behavior

![image](https://github.com/supabase/supabase/assets/113150193/cf4d2679-6c8b-475d-b99c-8984a520bd32)
the text inside the input box has low contrast, it should be more bright.

![image](https://github.com/supabase/supabase/assets/113150193/38aff11d-245e-4f24-8518-e412bf75c223)
the keymap "CTRL" has low contrast, it's hard to see it

![image](https://github.com/supabase/supabase/assets/113150193/4a4d19c8-a350-4919-841d-cfa1e3638662)
the keymap "ESC" in the button "discard" is almost invisible. increase contrast.

![image](https://github.com/supabase/supabase/assets/113150193/45c4690e-d0ef-4417-b187-86a248fcec10)
some of the "git difference" indicators in each line difficult to read the text.
the top text on the right query editor has low contrast, it should be incremented.

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of supabase-js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

some of the issues are easy to fix with CSS, but the SQL text editor is more difficult since I guess supabase is using an integration of vs code in some way (I don't know), but I guess it could be fixed by editing the theme.

hope it helps!

**Comment by Huilensolis** [comment]
I'm sorry for using the label "bug" for this issue, but I could not find any other issue template that was under the category of "this could improve, but its not a bug" : (

**Comment by bl-aire** [comment]
Hi,
Can I work on this?

**Comment by encima** [comment]
> Hi @Huilensolis @encima,
> 
> 
> 
> Can I work on this?

sure. you don't need permissions from anyone! go ahead and open a PR. thanks!

**Comment by Huilensolis** [comment]
How can I know if an issue is planned or not? If fhe issue has a tag assigned by te repo members, does that mean that it is planned to be implemented and I can open a pr? How does it work?

**Comment by bl-aire** [comment]
> > Hi @Huilensolis @encima,
> > 
> > 
> > 
> > Can I work on this?
> 
> sure. you don't need permissions from anyone! go ahead and open a PR. thanks!

Okay. Thank you

**Comment by encima** [comment]
> How can I know if an issue is planned or not? If fhe issue has a tag assigned by te repo members, does that mean that it is planned to be implemented and I can open a pr? How does it work?

@Huilensolis This is a good question and maybe has 2 answers:

## To choose issues you would like to work on: 
Anything labelled as `good-first-issue` or `help-wanted` are the best ones to start with

## To see if Supabase is working on it: 
This one is more difficult and is not immediately visible. Maybe the easiest way is to check if the issue has a PR open against it.

**Comment by encima** [comment]
@bl-aire 

we can help sure! But this is about a different issue, please open a new issue explaining the steps you followed, the versions you are running, the issue you are facing and any logs/errors that you experience.

**Comment by bl-aire** [comment]
> @bl-aire 
> 
> we can help sure! But this is about a different issue, please open a new issue explaining the steps you followed, the versions you are running, the issue you are facing and any logs/errors that you experience. 

Thank you!!
I'll do that now. Kinda new so I'm trying to understand supabase and its repository. 

With time, I'll get a hang of it
